gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -66,9 +68,15 @@ def main(args):
     test = args.test
     overwrite = args.overwrite
     max_af = args.max_af
-    partition_hint = args.partition_hint
+    partition_hint_when_creating_training_set = (
+        args.partition_hint_when_creating_training_set
+    )
+    partition_hint_when_counting_variants = args.partition_hint_when_counting_variants
+    partition_hint_for_sum_aggregators = args.partition_hint_for_sum_aggregators
```

Comment:
Not changing these names since I don't know if you will agree with my suggested names